src/mad_spark_alt/core/smart_orchestrator.py

```
@@ -169,6 +171,34 @@ async def run_qadi_cycle(
+                result.conclusion = await conclusion_synthesizer.synthesize_conclusion(
+                    problem_statement=problem_statement,
+                    ideas_by_phase=ideas_by_phase,
+                    context=context
+                )
+                
+                # Add cost of conclusion synthesis
+                if hasattr(result.conclusion, 'metadata') and 'llm_cost' in result.conclusion.metadata:
+                    total_llm_cost += result.conclusion.metadata['llm_cost']
+                    result.llm_cost = total_llm_cost
+                    
+            except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)